modules/dop/initialize.go

```diff
@@ -122,6 +122,8 @@ func (p *provider) Initialize(ctx servicehub.Context) error {
 
 	registerWebHook(bdl.Bdl)
 
+	deleteWebhook(bdl.Bdl)
```

Comment:
handle error correctly or not return error.

Reply:
done